nodes/transport/transport_final_demand_diesel.final_demand.ad

```diff
@@ -3,4 +3,4 @@
 - groups = [final_demand_group]
 - free_co2_factor = 0.0
 
-~ demand = EB("transport", "gas/diesel_oil")
+~ demand = EB("transport", "gas/diesel_oil") - EB("non-specified (transport)", "gas/diesel_oil") - EB("pipeline transport", "gas/diesel_oil")
```

Comment:
@RobTerwel , this means that we are ignoring any energy that is reported under `non-specified (transport)` and `pipeline transport`, right? Could you think of a fix for this?